net-analyzer/graphite-web/files/graphite-web-1.1.3-fhs-paths.patch

```diff
@@ -36,7 +36,7 @@ Install FHS-style paths
  #INDEX_FILE = '/opt/graphite/storage/index'     # Search index file
 +CONF_DIR = '@GENTOO_PORTAGE_EPREFIX@/etc/graphite-web'
 +STORAGE_DIR = '@GENTOO_PORTAGE_EPREFIX@/var/lib/carbon'
-+STATIC_ROOT = '@GENTOO_PORTAGE_EPREFIX@/usr/share/graphite-web'
++STATIC_ROOT = '@GENTOO_PORTAGE_EPREFIX@/usr/share/graphite-web/webapp/content'
```

Comment:
nice catch, seems I got this fixed in my setup with puppet, but didn't notice the default was wrong

Reply:
On my setup, nginx is serving the static files so I don't think it matters. I assume it is not used unless graphite itself is serving those:
> Alternatively, static files can be served directly by the Graphite webapp if you install the whitenoise Python package.

Reply:
I remember having issues with it, because my graphite-web instance is running on a different machine than the machine running nginx, so I had to forward proxy it because I couldn't get graphite-web to do it.